packages/webpack/src/loader.ts

```diff
@@ -20,14 +20,18 @@ interface LoaderContext {
   async: () => (err: unknown, result?: string) => void;
 }
 
+// Workaround to avoid consumers of the loader/webpack plugin having to add
```

Comment:
Is this a workaround, or just a best practice thing to do?
Seems safer to use the loader from this dependency than the app's one.

Reply:
I copied the idea [from vanilla](https://github.com/vanilla-extract-css/vanilla-extract/blob/c59aee20079d66e8abe4ca23e0a2d673940139a8/packages/webpack-plugin/src/loader.ts#L16), but in that case the virtual file loader is inside the plugin package, so it's not really a dependency anyway. I think the idea transfers across though, and I agree that it's safer to the webpack plugin's dependency rather than the app's one.

I can remove the comment if needed, was mainly a comment for myself anyway.